docs/coordinates/performance.inc.rst

```diff
@@ -56,3 +56,157 @@ properties::
 
 ..
   EXAMPLE END
+
+
+Improving Performance for Arrays of ``obstime``
+-----------------------------------------------
+
+The most expensive operations when transforming between observer-dependent coordinate
+frames (e.g. ``AltAz``) and sky-fixed frames (e.g. ``ICRS``) are the calculation
+of the orientation and position of Earth.
+
+If |skycoord| instances with a large ``obstime`` array are transformed,
```

Comment:
Maybe "If |skycoord| instances are transformed for a large number of closely spaced ``obstime``"

Reply:
done